src/lib/migration.js

```diff
@@ -164,13 +164,13 @@ const dropAndRecreateEnum = async (
   enumValues = [],
   enumType = 'text',
 ) => {
-  await queryInterface.sequelize.query(`  
+  await queryInterface.sequelize.query(`
   -- rename the existing type
   ALTER TYPE "${enumName}" RENAME TO "${enumName}_old";
   -- create the new type
-  CREATE TYPE "${enumName}" AS ENUM();`, { transaction });
-
-  await addValuesToEnumIfTheyDontExist(queryInterface, transaction, enumName, enumValues);
+  CREATE TYPE "${enumName}" AS ENUM(
+    ${Object.values(enumValues).map((enumValue) => `'${enumValue}'`).join(',\n')}
```

Comment:
I think it's fine (but strange) to pass an array to `Object.values`, because you'll still end up mapping over the array values. If `enumValues` is an array (`FEATURE_FLAGS` is at least), maybe just `enumValues.map` is enough.